addon/components/frost-application-bar.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember'
+import layout from '../templates/components/frost-application-bar'
```

Comment:
app bar is part of navigation? The nav component was only supposed to be the modal dialog to allow for it to be launched from any kind of component a consumer may want